ascentcsv1.csv

```diff
@@ -1,111 +1,110 @@
-Linked Doors Intro Missile,0,0,0x1c0948,0,0,0,open,0,0,0,Linked Doors Intro Missile,0,0,
-Volonus Weapon Upgrade,0,0,0x1c11d6,0,0,0,open,0,0,0,Volonus Weapon Upgrade,0,0,
-Generators Core Super,0,0,0x1c09b6,0,0,0,open,0,0,0,Generators Core Super,0,0,
-Speed Booster,0,0,0x1c46b3,0,0,0,open,0,0,0,Speed Booster,0,0,
-Living Presence Energy,0,0,0x1c12fe,0,0,0,open,0,0,0,Living Presence Energy,0,0,
-Chain Reaction Spark Upgrade,0,0,0x1c1db0,0,0,0,open,0,0,0,Chain Reaction Spark Upgrade,0,0,
-Grapple Beam,0,0,0x1c0824,0,0,0,open,0,0,0,Grapple Beam,0,0,
-Easy Choice Super,0,0,0x1c01e2,0,0,0,open,0,0,0,Easy Choice Super,0,0,
-Target Jump Reserve,0,0,0x1c0b24,0,0,0,open,0,0,0,Target Jump Reserve,0,0,
-Closing the Loop Energy Tank,0,0,0x1c11b4,0,0,0,open,0,0,0,Closing the Loop Energy Tank,0,0,
-Burning Curiosity Missile,0,0,0x1c0866,0,0,0,open,0,0,0,Burning Curiosity Missile,0,0,
-Moisture Challenge Weapon Upgrade,0,0,0x1c1290,0,0,0,open,0,0,0,Moisture Challenge Weapon Upgrade,0,0,
-Expand Shaft R Missile,0,0,0x1c0af2,0,0,0,open,0,0,0,Expand Shaft R Missile,0,0,
-Slug Missile,0,0,0x1c0642,0,0,0,open,0,0,0,Slug Missile,0,0,
-Jail Energy,0,0,0x1c04d8,0,0,0,open,0,0,0,Jail Energy,0,0,
-Fullmetal Cor. Missile,0,0,0x1c03bc,0,0,0,open,0,0,0,Fullmetal Cor. Missile,0,0,
-Trap Weapon Upgrade,0,0,0x1c0c44,0,0,0,open,0,0,0,Trap Weapon Upgrade,0,0x1c133c,
-They Excite Me Missile,0,0,0x1c0c36,0,0,0,open,0,0,0,They Excite Me Missile,0,0,
-Expand Shaft L Energy,0,0,0x1c0afa,0,0,0,open,0,0,0,Expand Shaft L Energy,0,0,
-Mount Top Energy,0,0,0x1c4365,0,0,0,open,0,0,0,Mount Top Energy,0,0,
-Big Purp Missile,0,0,0x1c0b66,0,0,0,open,0,0,0,Big Purp Missile,0,0,
-Super Get,0,0,0x1c0aca,0,0,0,open,0,0,0,Super Get,0,0,
-Horseshoe Missile,0,0,0x1c0dd8,0,0,0,open,0,0,0,Horseshoe Missile,0,0,
-Energizer Energy,0,0,0x1c0e0a,0,0,0,open,0,0,0,Energizer Energy,0,0,
-Boost Ball,0,0,0x1c0876,0,0,0,open,0,0,0,Boost Ball,0,0,
-Toxic Air Avoid Super,0,0,0x1c0d30,0,0,0,open,0,0,0,Toxic Air Avoid Super,0,0,
-Toxic Lab East Super,0,0,0x1c0fb8,0,0,0,open,0,0,0,Toxic Lab East Super,0,0,
-Mixing Knowledge Missile,0,0,0x1c0cb0,0,0,0,open,0,0,0,Mixing Knowledge Missile,0,0,
-Explorer Reserve,0,0,0x1c0c04,0,0,0,open,0,0,0,Explorer Reserve,0,0,
-Nestroid Climb Super,0,0,0x1c0c1c,0,0,0,open,0,0,0,Nestroid Climb Super,0,0,
-Gate Prologue Missile,0,0,0x1c4389,0,0,0,open,0,0,0,Gate Prologue Missile,0,0,
-Reflects Trail Weapon Upgrade,0,0,0x1c446f,0,0,0,open,0,0,0,Reflects Trail Weapon Upgrade,0,0,
-Interchange Missile,0,0,0x1c4503,0,0,0,open,0,0,0,Interchange Missile,0,0,
-Bio Kamers Super,0,0,0x1c11c2,0,0,0,open,0,0,0,Bio Kamers Super,0,0,
-Endgame Blockade Spark Upgrade,0,0,0x1c0ffe,0,0,0,open,0,0,0,Endgame Blockade Spark Upgrade,0,0,
-Spazer,0,0,0x1c0ee2,0,0,0,open,0,0,0,Spazer,0,0x1c113c,
-Spazer Fountain Missile,0,0,0x1c0ef4,0,0,0,open,0,0,0,Spazer Fountain Missile,0,0,
-Gettin Around It Varia,0,0,0x1c1154,0,0,0,open,0,0,0,Gettin Around It Varia,0,0x1c0eee,
```

Comment:
In addition to adding a "zone" option to this csv I also removed this location. This was breaking major/minor splits because we need t have the same number of majors and major locations (previously the minors were wrong so this bug went unnoticed).

The short term fix is to just duplicate whatever item is at "Taste the Rainbow Varia", but for now this is varia in every seed.